demo/src/react-admin/modules/user-group/const/user-group.const.tsx

```diff
@@ -0,0 +1,148 @@
+		label: Config.getConfig().services.i18n.t(
+			'Admin meemoo'
+		),
+	},
+	{
+		name: UserGroups.CP_ADMIN,
+		label: Config.getConfig().services.i18n.t(
+			'CP admin'
+		),
+	},
+	{
+		name: UserGroups.VISITOR,
+		label: Config.getConfig().services.i18n.t(
+			'Visitor'
+		),
+	},
+	{
+		name: UserGroups.KIOSK_VISITOR,
+		label: Config.getConfig().services.i18n.t(
+			'Kiosk visitor'
+		),
+	},
+]
+
+export const UserGroupTableColumns = (
+	userGroups: UserGroupArchief[],
+	updateUserGroup: (groupId: string, permissionId: string, value: boolean) => void,
+): (Column<PermissionData> & UseSortByColumnOptions<PermissionData>)[] => [
+	{
+		Header: '',
+		accessor: 'label',
+		disableSortBy: true,
+	},
+	...GET_USER_GROUPS().map((groupLabel) => {
+		const group = userGroups.find((group) => group.name === groupLabel.name);
+
+		return {
+			Header: groupLabel.label || group?.name,
+			id: `${group?.name}-${group?.id}`,
+			accessor: (row: PermissionData) => row.name ,
+			disableSortBy: true,
+			Cell: ({row}: PermissionRow) => {
+				const isChecked = group?.permissions
+					? !!group?.permissions.find((permission: PermissionData) => permission.id === row.original.id)
+					: false;
+
+				return <Checkbox
+					checked={isChecked}
+					value={`${group?.name}-${row.original.name}`}
+					onChange={() => updateUserGroup(String(group?.id), row.original.id, !isChecked)}
+				/>
+			},
+		}
+	}),
+]
+
+export const GET_SPECIAL_USER_GROUPS: () => Partial<UserGroup>[] = () => [
+	{
+		label: Config.getConfig().services.i18n.t(
+			'admin/menu/components/menu-edit-form/menu-edit-form___niet-ingelogde-gebruikers'
+		),
+		id: SpecialPermissionGroups.loggedOutUsers,
+	},
+	{
+		label: Config.getConfig().services.i18n.t(
+			'admin/menu/components/menu-edit-form/menu-edit-form___ingelogde-gebruikers'
+		),
+		id: SpecialPermissionGroups.loggedInUsers,
+	},
+];
+
+export const GET_USER_GROUP_OVERVIEW_TABLE_COLS: () => FilterableColumn[] = () => [
+	{
+		id: 'label',
+		label: Config.getConfig().services.i18n.t('admin/user-groups/user-group___label'),
+		sortable: true,
+		visibleByDefault: true,
+		dataType: 'string',
+	},
+	{
+		id: 'description',
+		label: Config.getConfig().services.i18n.t('admin/user-groups/user-group___beschrijving'),
+		sortable: true,
+		visibleByDefault: true,
+		dataType: 'string',
+	},
+	{
+		id: 'created_at',
+		label: Config.getConfig().services.i18n.t('admin/user-groups/user-group___aangemaakt-op'),
+		sortable: true,
+		visibleByDefault: true,
+		filterType: 'DateRangeDropdown',
+		dataType: 'dateTime',
+	},
+	{
+		id: 'updated_at',
+		label: Config.getConfig().services.i18n.t('admin/user-groups/user-group___aangepast-op'),
+		sortable: true,
+		visibleByDefault: true,
+		filterType: 'DateRangeDropdown',
+		dataType: 'dateTime',
+	},
+	{
+		id: 'actions',
+		tooltip: Config.getConfig().services.i18n.t('admin/user-groups/user-group___acties'),
+		visibleByDefault: true,
+	},
+];
+
+export enum SpecialUserGroup {
```

Comment:
this is duplicated in 2 files:
![image](https://user-images.githubusercontent.com/1710840/168869282-e8a534f6-f388-4fb4-870b-738fdfa5c4e2.png)

Reply:
Hier waren idd een aantal duplicates. Fixed.